Dockerfile

```diff
@@ -39,8 +39,8 @@ ENV AUTHENTICATION_SERVICE_URL=http://identity.local.cshr.digital:9443 \
   PORT=3001 \
   SESSION_SECRET=topsecret
 
-# TODO(tav): Create an actual user so that we don't run this as root
```

Comment:
Leave the note in there until the root issue is addressed?

Reply:
Maybe it's cos I'm a cowboy, but I don't think this is such an issue when running inside containers.